bitnami/kafka/templates/jmx-configmap.yaml

```diff
@@ -50,6 +50,10 @@ data:
         name: java_lang_$1_$4_$3_$2
       - pattern: java.lang<type=(.+), name=(.+)><>(\w+)
         name: java_lang_$1_$3_$2
+      - pattern: java.lang<name=(.*), type=GarbageCollector><>CollectionCount
+        type: COUNTER
+      - pattern: java.lang<name=(.*), type=GarbageCollector><>CollectionTime
+        type: GAUGE
```

Comment:
What do you think about adding a generic way to add custom patterns? I mean, adding a new parameter that allows setting your custom patterns in the `values.yaml` so we provide more freedom to users to customize the metrics to be exposed in the JMX exporter.

Reply:
Sure, that sounds great. I understand that currently the only other option is to supply a full custom JMX exporter config, which is not awesome.
I'm not much of a Helm guy so I won't be helpful here, but I can make sense of JMX stuff...

Reply:
Can't you extend the current value for `metrics.jmx.config`? I mean, install the chart using the `values.yaml` below:

```yaml
metrics:
  jmx:
    config: |-
      jmxUrl: service:jmx:rmi:///jndi/rmi://127.0.0.1:5555/jmxrmi
      lowercaseOutputName: true
      lowercaseOutputLabelNames: true
      ssl: false
      {{- if .Values.metrics.jmx.whitelistObjectNames }}
      whitelistObjectNames: ["{{ join "\",\"" .Values.metrics.jmx.whitelistObjectNames }}"]
      {{- end }}
      rules:
        - pattern: java.lang<name=(.*), type=GarbageCollector><>CollectionCount
          type: COUNTER
        - pattern: java.lang<name=(.*), type=GarbageCollector><>CollectionTime
          type: GAUGE
```